src/ServerTelemetryChannel/Managed/Shared/SamplingTelemetryProcessor.cs

```diff
@@ -125,6 +126,17 @@ public string IncludedTypes
         }
 
         /// <summary>
+        /// Gets or sets a value indicating whether or not we should calculate sampling based on the Context.User.Id field.
+        /// Default is false.
+        /// 
+        /// <remarks>
+        /// Calculation of the sampling score based on user_Id field is disabled by default to ensure successful correlation across many 
+        /// services on operation_Id field.
+        /// </remarks>
+        /// </summary>
+        public bool AllowSamplingBasedOnUserId { get; set; }
```

Comment:
`Allow` is not the best verb here. Maybe `Calculate` like in `CalculateSamplingScoreFromUserId` or `Use` like in `UseUserIdSamplingScore`